json/src/main/kotlin/ktx/json/serializer.kt

```diff
@@ -0,0 +1,40 @@
+package ktx.json
+
+import com.badlogic.gdx.utils.Json
+import com.badlogic.gdx.utils.JsonValue
+
+/**
+ * Wrapping interface around [com.badlogic.gdx.utils.Json.Serializer].
+ * Provides null-safety for the methods.
+ */
+interface Serializer<T> : Json.Serializer<T> {
```

Comment:
Add `Json` Prefix to class name, since `Serializer` is a bit too vague and clashes with nested LibGDX class name. I think the classes should be named `JsonSerializer` and `ReadOnlyJsonSerializer`.

Reply:
Sorry, a bit missed this one.

Reply:
It's OK. :)